ndcube/utils/wcs.py

```diff
@@ -209,13 +209,18 @@ def _wcs_slicer(wcs, missing_axis, item):
     # what was inputed so we had a reverse missing_axis.
     extra_coords_list = [] # initiating new list
     for slice_element in item_: # check item_ slices for dropped axes if any
-        if (slice_element.stop - slice_element.start) == 1:
-           extra_coords_list.append(slice_element)
-        else:
-           pass
+        for _bool in missing_axis:
+            if not _bool:
+                if index is not 1: 
```

Comment:
What is this line for?  I'm not sure it's needed or correct as index is not redefined after the ```if elif``` statements further up...?

Reply:
Sorry, my bad. Was confused about how to checking for the boolean value of the `missing_axis`...